### PR TITLE
Feature/project improvements

### LINKED ISSUE #30: Add a LocalValetDriver.php file
For easier configuration when using [Valet](https://laravel.com/docs/8.x/valet) to serve local vhosts, it could be useful to add a `LocalValetDriver.php` file extending the default `WordPressValerDriver`.

A simple example to serve the content of the `web/` directory:
```php
class LocalValetDriver extends WordPressValetDriver
{
	/**
	 * Determine if the driver serves the request.
	 *
	 * @param  string  $sitePath
	 * @param  string  $siteName
	 * @param  string  $uri
	 * @return bool
	 */
	public function serves($sitePath, $siteName, $uri)
	{
		$sitePath .= '/web';
		return parent::serves($sitePath, $siteName, $uri);
	}

        public function isStaticFile($sitePath, $siteName, $uri)
	{
		$sitePath .= '/web';
		return parent::isStaticFile($sitePath, $siteName, $uri);
	}

	/**
	 * Get the fully resolved path to the application's front controller.
	 *
	 * @param  string  $sitePath
	 * @param  string  $siteName
	 * @param  string  $uri
	 * @return string
	 */
	public function frontControllerPath($sitePath, $siteName, $uri)
	{
		$sitePath .= '/web';
		return parent::frontControllerPath(
			$sitePath,
			$siteName,
			$this->forceTrailingSlash($uri)
		);
	}
}
```

We could use this to redirect the `web/wp-content/uploads` requests to the preproduction or production environment:

```php
public function frontControllerPath($sitePath, $siteName, $uri)
{
    if (false !== strpos($uri, '/wp-content/uploads/') && $remote_host = getenv('REMOTE_HOST')) {
        header('Location: '. 'http://' . $remote_host . '/' . $uri ); 
        die;
    }
    
    // ...
}
```

### LINKED ISSUE #31: Add stoutlogic/acf-builder to the Composer dependencies when using ACF
For easier ACF fields management, we should always use the [stoutlogic/acf-builder](https://github.com/StoutLogic/acf-builder) package.

### LINKED ISSUE #34: Update default PHP version to ^7.3
The title says it all.

### LINKED ISSUE #36: Add a Twig comment debugger in debug mode and/or local env
See https://github.com/djboris88/twig-commented-include.

### LINKED ISSUE #37: Add a Twig alias for the svg folder
In `TwigManager.php` : 

```php
public function run() {
        // ...
	add_filter( 'timber/loader/loader', array( $this, 'add_svg_path' ), 10, 1 );
}

/**
 * Add an alias for the SVG folder.
 *
 * @param \Twig\Loader\FilesystemLoader $fs The loader to extend.
 * @return \Twig\Loader\FilesystemLoader
 */
public function add_svg_path( \Twig\Loader\FilesystemLoader $fs ) {
	$fs->addPath( get_template_directory() . '/static/svg', 'svg' );
	return $fs;
}
```

### LINKED ISSUE #38: Add a `[data-breakpoint]` attribute on the `<body>` tag

### LINKED ISSUE #39: Update the list of plugins in the prompt
- [x] Replace Yoast by RankMath
- [x] Set ACF, WP Rocket, Classic editor, WP Rocket selected by default

### LINKED ISSUE #40: Update the version of ACF
In the `composer.json` update the package definition.

### LINKED ISSUE #41: Update the WordPress version
We should only limit the major version to `5` : 

```diff
- "johnpbloch/wordpress": "^5.2",
+ "johnpbloch/wordpress": "^5",
```

### LINKED ISSUE #42: Add the .htacces.example file to the project's readme

### PR DESCRIPTION
**Pull request by perruche**
## Added
- Add a local Valet Driver (Fix #30)
- Add a [data-breakpoint] attribute on `<body>` (Fix #38)
- Add Twig debug comments (Fix #36)
- Add a Twig alias for the svg directory (Fix #37)
- Add `stoutlogic/acf-builder` package (Fix #31)

## Updated
- Update default PHP version to ^7.3 (Fix #34)
- Update WordPress version to use latest `5.*` (Fix #41)
- Update ACF version (Fix #40)
- Update README to include `.htaccess.example` (Fix #42)

### CODE REVIEW

File: template/composer.json [hunk 1]
```diff
@@ -23,13 +23,15 @@
...
     <%_ } _%>
-    <%_ if (yoastSeo) { _%>
-    "yoast/wordpress-seo": "^12.9",
+    <%_ if (rankMath) { _%>
+    "wpackagist-plugin/seo-by-rank-math": "^1.0",
     <%_ } _%>
     <%_ if (acf) { _%>
-    "advanced-custom-fields/advanced-custom-fields-pro": "^5.0.0",
+    "advanced-custom-fields/advanced-custom-fields-pro": "5.*",
+    "stoutlogic/acf-builder": "^1.10",
```
Comment (titouanmathis) [inline template/composer.json:31]: 👍

File: template/web/wp-content/themes/<%= slug %>/app/Managers/ACFManager.php [hunk 2]
```diff
@@ -2,83 +2,37 @@
...
-							'value'    => 'post',
-						),
-					),
-				),
-			)
-		);
+		$example_group = new FieldsBuilder( 'example_group' );
+		$example_group->addImage( 'image' )
+			->setLocation( 'post_type', '==', 'post' );
+		acf_add_local_field_group( $example_group->build() );
```
Comment (titouanmathis) [inline template/web/wp-content/themes/<%= slug %>/app/Managers/ACFManager.php:36]: 😍

File: template/web/wp-content/themes/<%= slug %>/app/Managers/TwigManager.php [hunk 3]
```diff
@@ -29,4 +33,48 @@ public function add_twig_template_from_string( \Twig\Environment $twig ) {
...
+	 * @param array  $data data.
+	 * @param string $file name.
+	 * @return string
+	 */
+	public function add_twig_template_render_comments( $output, $data, $file ) {
+		if ( getenv( 'APP_DEBUG' ) === 'false' ) {
+			return $output;
+		}
+
+		return "\n<!-- Begin output of '" . $file . "' -->\n" . $output . "\n<!-- / End output of '" . $file . "' -->\n";
```
Comment (titouanmathis) [inline template/web/wp-content/themes/<%= slug %>/app/Managers/TwigManager.php:66]: Do we need both `add_twig_template_include_comments` and `add_twig_template_render_comments`? 🤔

**Review by titouanmathis — APPROVED** [review]

File: template/web/wp-content/themes/<%= slug %>/app/Managers/TwigManager.php (hunk 3 above)
Comment (perruche) [reply template/web/wp-content/themes/<%= slug %>/app/Managers/TwigManager.php:66]: While it's close in behavior, i think this adding usefull information: 

`add_twig_template_include_comments` is adding comments for every included/embed files. 
`add_twig_template_render_comments` is adding one comment after each `Timber::render`, so you will get the comment `<!-- Begin output of pages/single.twig -->` at the top of the DOM which is super usefull (espacially if it's a more specific WP template)

**Pull request merged**